code-quality/checkstyle/checkstyle-suppressions.xml

```diff
@@ -8,6 +8,8 @@
     <suppress files=".*Application.java" checks="HideUtilityClassConstructor"/>
 
     <suppress files="[\\/]model[\\/].*.java" checks="JavadocMethod"/>
+    <suppress files="[\\/]config[\\/].*.java" checks="JavadocMethod"/>
```

Comment:
Better to disable such checks in the code if it is possible (add TODO and issue in repo to solve it)